### ISSUE TITLE
Error when weight is None

### ISSUE THREAD

**Issue by Gerenuk** [CalculatedContent/WeightWatcher#118]
WeightWatcher fails when `weight` is None as it is for `BatchNorm2d(num, affine=False)`
```
File .../lib/python3.10/site-packages/weightwatcher/weightwatcher.py:311, in WWLayer.get_weights_and_biases(self)
    309 if self.framework == FRAMEWORK.PYTORCH:
    310     if hasattr(self.layer, 'weight'):
--> 311         w = [np.array(self.layer.weight.data.clone().cpu())]
    312         if self.the_type==LAYER_TYPE.CONV2D:
    313             weights = w[0]
```

**Comment by charlesmartin14** [comment]
Can you send me a version of the model with this ?
It should be skipping this layer

**Comment by charlesmartin14** [comment]
Closing this issue since it can not be reproduced

**Comment by MoritzBoe** [comment]
I had the same problem in the current version 0.7.1.5.
Here is a minimal example with pytorch 2.0
```
from torch import nn
import weightwatcher as ww

class NormNet(nn.Module):
    def __init__(self):
        super().__init__()
        self.layers = nn.Sequential(
            nn.Linear(10*10, 512),
            nn.BatchNorm1d(512, affine=False),
            nn.Linear(512, 2)
        )
    def forward(self, x):
        out = self.layers(x)
        return out

model = NormNet()

watcher = ww.WeightWatcher(model=model)
details = watcher.analyze()
```

Normalization layers in Pytorch have the weight attribute, but it is set to none for affine=False, which is standard for the instancenorm layer.

I fixed it by modifying the get_weights_and_biases functions in the PyTorchLayer class in weightwatcher.py like:
```
if not hasattr(self.layer, 'weight'):
            return has_weights, weights, has_biases, biases  
        if self.layer.weight == None:
            return has_weights, weights, has_biases, biases  
```
